dagster_sqlmesh/controller/base.py

```diff
@@ -439,13 +443,15 @@ def setup_with_config(
         config: SQLMeshContextConfig,
         context_factory: ContextFactory[ContextCls] = DEFAULT_CONTEXT_FACTORY,
         log_override: logging.Logger | None = None,
+        translator: SQLMeshDagsterTranslator = SQLMeshDagsterTranslator(),
```

Comment:
```suggestion
        translator: SQLMeshDagsterTranslator | None = None
```

Do this and set the value to the instance later in the factory method.